profile/src/test/resources/test-profiles/valid/simple-data-constraints-nested-7level.json

```diff
@@ -1,334 +0,0 @@
-                            },
-                            {
-                              "allOf": [
-                                {
-                                  "field": "field1",
-                                  "is": "null"
-                                },
-                                {
-                                  "field": "field1",
-                                  "is": "ofType",
-                                  "value": "ISIN"
-                                },
-                                {
-                                  "field": "field1",
-                                  "is": "inSet",
-                                  "values": ["1", 2]
-                                },
-                                {
-                                  "field": "field1",
-                                  "is": "ofType",
-                                  "value": "string"
-                                },
-                                {
-                                  "field": "field2",
-                                  "is": "greaterThan",
-                                  "value": 43
-                                },
-                                {
-                                  "field": "field2",
-                                  "is": "lessThan",
-                                  "value": 78
-                                },
-                                {
-                                  "field": "field2",
-                                  "is": "greaterThanOrEqualTo",
-                                  "value": 44
-                                },
-                                {
-                                  "field": "field2",
-                                  "is": "lessThanOrEqualTo",
-                                  "value": 77
-                                },
-                                {
-                                  "field": "field2",
-                                  "is": "granularTo",
-                                  "value": 0.1
-                                },
-                                {
-                                  "field": "field2",
-                                  "is": "equalTo",
-                                  "value": 0.004003
-                                },
-                                {
-                                  "field": "field1",
-                                  "is": "equalTo",
-                                  "value": "0.004003"
-                                }
-                              ]
-                            }
-                          ]
-                        }
-                      ]
-                    }
-                  ]
-                }
-              ]
-            }
-          ]
-        }
-      ]
-    }
-  ],
-  "fields": [
-    {
-      "name": "field1",
-      "formatting": "%ggg"
-    },
-    {
-      "name": "field2"
-    }
-  ]
-}
```

Comment:
We probably want to keep this unless you think it will be hard to maintain in future?

Reply:
I'm not sure what this file really gives use. Considering we already have a 2 level test, it is also a bit of a nightmare to maintain.